python/CHANGES.rst

```diff
@@ -1,6 +1,12 @@
 Changelog for tchannel.py
 =========================
 
+0.9.2 (unreleased)
```

Comment:
The version number should be finalized by the person who actually cut the release.